fixes: default template of Redis Insight

rohittiwari-dev opened the pull request:
this PR refers the issue generated that , caused by untested template issue no : #7166

Redis Insight is inaccessible when installed with default configuration until manually change the test url

## Changes
- Change the test health check url for template to work

## Issues
- fix #7166

rohittiwari-dev committed:
fixes default template of Redis Insight

this PR refers the issue generated that , caused by untested template issue no : #7166

Redis Insight is inaccessible when installed with default configuration until manually change the test url

ShadowArcanist commented:
Existing template already works fine and the current health check is passing:

https://github.com/user-attachments/assets/494fc25d-e570-4db0-b727-afc495d7b31f

rohittiwari-dev commented:
> Existing template already works fine and the current health check is passing:
> 
>  redis-insight.mov

most probably you are installing the service in same server where coolify is installed not on the second server attached to coolify have you tested it ? because i got to have same issue as stated in the closed issue

ShadowArcanist commented:
> > Existing template already works fine and the current health check is passing:
> > redis-insight.mov
> 
> most probably you are installing the service in same server where coolify is installed not on the second server attached to coolify have you tested it ? because i got to have same issue as stated in the closed issue

On the attached video at 0:05 timestamp you can see I am choosing my remote server than the server where Coolify is running.

I just deploying the service on server where Coolify is running and I got the health check failing, so it's not an issue with the template itself but something else.

My Coolify running on arm64 server and remote server is amd64, can you share the cpu arch of your servers? (would be helpful for debugging this issue further)

rohittiwari-dev commented:
<img width="929" height="487" alt="image" src="https://github.com/user-attachments/assets/8f22d19f-5b0c-460b-82be-a703187f9799" />


here it is

ShadowArcanist commented:
> <img alt="image" width="929" height="487" src="https://private-user-images.githubusercontent.com/137717184/512145954-8f22d19f-5b0c-460b-82be-a703187f9799.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjI3Nzg0MzcsIm5iZiI6MTc2Mjc3ODEzNywicGF0aCI6Ii8xMzc3MTcxODQvNTEyMTQ1OTU0LThmMjJkMTlmLTViMGMtNDYwYi04MmJlLWE3MDMxODdmOTc5OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTExMFQxMjM1MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03Y2RiMDgyOGZhMzY2ZmM4ZTIxMTlkMTI0NWQ0MmYyN2JmNmE0MTdhNjlhZDM3MDA3Y2U4MGFlMmI3OGJkZGFmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ghcc6GiZ066r9TejH7xnvXXG3nklYrym3-trb1g0EGM">
> 
> here it is

So the healthcheck working or failing on this server (x86) ?

rohittiwari-dev commented:
> > <img alt="image" width="929" height="487" src="https://private-user-images.githubusercontent.com/137717184/512145954-8f22d19f-5b0c-460b-82be-a703187f9799.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjI3Nzg0MzcsIm5iZiI6MTc2Mjc3ODEzNywicGF0aCI6Ii8xMzc3MTcxODQvNTEyMTQ1OTU0LThmMjJkMTlmLTViMGMtNDYwYi04MmJlLWE3MDMxODdmOTc5OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTExMFQxMjM1MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03Y2RiMDgyOGZhMzY2ZmM4ZTIxMTlkMTI0NWQ0MmYyN2JmNmE0MTdhNjlhZDM3MDA3Y2U4MGFlMmI3OGJkZGFmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ghcc6GiZ066r9TejH7xnvXXG3nklYrym3-trb1g0EGM">
> > here it is
> 
> So the healthcheck working or failing on this server (x86) ?

it is failing
but when i changed it , it worked and is now accessible on internet on custom domain as well

ShadowArcanist commented:
> > > <img alt="image" width="929" height="487" src="https://private-user-images.githubusercontent.com/137717184/512145954-8f22d19f-5b0c-460b-82be-a703187f9799.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjI3Nzg0MzcsIm5iZiI6MTc2Mjc3ODEzNywicGF0aCI6Ii8xMzc3MTcxODQvNTEyMTQ1OTU0LThmMjJkMTlmLTViMGMtNDYwYi04MmJlLWE3MDMxODdmOTc5OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTExMFQxMjM1MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03Y2RiMDgyOGZhMzY2ZmM4ZTIxMTlkMTI0NWQ0MmYyN2JmNmE0MTdhNjlhZDM3MDA3Y2U4MGFlMmI3OGJkZGFmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ghcc6GiZ066r9TejH7xnvXXG3nklYrym3-trb1g0EGM">
> > > here it is
> > 
> > 
> > So the healthcheck working or failing on this server (x86) ?
> 
> it is failing but when i changed it , it worked and is now accessible on internet on custom domain as well

Thanks!

This is very strange issue because the default healthcheck we have on template is failing for you on x86 but it's passing for me on x86.

rohittiwari-dev commented:
> > > > <img alt="image" width="929" height="487" src="https://private-user-images.githubusercontent.com/137717184/512145954-8f22d19f-5b0c-460b-82be-a703187f9799.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjI3Nzg0MzcsIm5iZiI6MTc2Mjc3ODEzNywicGF0aCI6Ii8xMzc3MTcxODQvNTEyMTQ1OTU0LThmMjJkMTlmLTViMGMtNDYwYi04MmJlLWE3MDMxODdmOTc5OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTExMFQxMjM1MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03Y2RiMDgyOGZhMzY2ZmM4ZTIxMTlkMTI0NWQ0MmYyN2JmNmE0MTdhNjlhZDM3MDA3Y2U4MGFlMmI3OGJkZGFmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ghcc6GiZ066r9TejH7xnvXXG3nklYrym3-trb1g0EGM">
> > > > here it is
> > > 
> > > 
> > > So the healthcheck working or failing on this server (x86) ?
> > 
> > 
> > it is failing but when i changed it , it worked and is now accessible on internet on custom domain as well
> 
> Thanks!
> 
> This is very strange issue because the default healthcheck we have on template is failing for you on x86 but it's passing for me on x86.

yes it felt me strange too i thought it was a version issue with coolify i updated it to latest , then also it did not worked

djsisson commented:
<img width="438" height="372" alt="image" src="https://github.com/user-attachments/assets/142da26e-81e4-4600-855f-85c26ad55dc8" />

its maybe an ipv6 issue with node, if localhost returns ::1 it fails

rohittiwari-dev commented:
Anyway the address 0.0.0.0 worked for me and most probably because it is whilecard address not loopback and will work in most cases if it  has no issue while testing we can go with it
